noxfiles/dev_nox.py

```diff
@@ -44,3 +44,29 @@ def quickstart(session: nox.Session) -> None:
     build(session, "dev")
     session.notify("teardown")
     run_infrastructure(datastores=["mongodb", "postgres"], run_quickstart=True)
+
+
+@nox.session()
+@nox.parametrize(
+    "script_name",
+    [
+        nox.param("example_script", id="example_script"),
+    ],
+)
+def run_script(session: nox.Session, script_name: str) -> None:
+    """
+    Run a script from the scripts/ directory. This command will not run
+    the server automatically.
+    """
+    posargs = session._runner.posargs
+    posargs.remove(script_name)
```

Comment:
this throws an error if it isn't present, is that intentional? I couldn't get it to run like this